src/instructlab/lab.py

```diff
@@ -79,1296 +95,21 @@ def __init__(self, config_obj: config.Config):
-            server_queue.join_thread()
-
-
-@cli.command()
-@click.option(
-    "--repository",
-    default="instructlab/merlinite-7b-lab-GGUF",  # TODO: add to config.yaml
-    show_default=True,
-    help="Hugging Face repository of the model to download.",
-)
-@click.option(
-    "--release",
-    default="main",  # TODO: add to config.yaml
-    show_default=True,
-    help="The git revision of the model to download - e.g. a branch, tag, or commit hash.",
-)
-@click.option(
-    "--filename",
-    default=basename(config.DEFAULT_MODEL_PATH),
-    show_default=True,
-    help="Name of the model file to download from the Hugging Face repository.",
-)
-@click.option(
-    "--model-dir",
-    default=dirname(config.DEFAULT_MODEL_PATH),
-    show_default=True,
-    help="The local directory to download the model files into.",
-)
-@click.option(
-    "--hf-token",
-    default="",
-    envvar="HF_TOKEN",
-    help="User access token for connecting to the Hugging Face Hub.",
-)
-@click.pass_context
-def download(ctx, repository, release, filename, model_dir, hf_token):
-    """Download the model(s) to train"""
-    click.echo(f"Downloading model from {repository}@{release} to {model_dir}...")
-    if hf_token == "" and "instructlab" not in repository:
-        raise ValueError(
-            """HF_TOKEN var needs to be set in your environment to download HF Model.
-            Alternatively, the token can be passed with --hf-token flag.
-            The HF Token is used to authenticate your identity to the Hugging Face Hub."""
-        )
-    try:
-        if ctx.obj is not None:
-            hf_logging.set_verbosity(ctx.obj.config.general.log_level.upper())
-        files = list_repo_files(repo_id=repository, token=hf_token)
-        if any(".safetensors" in string for string in files):
-            if not os.path.exists(os.path.join(model_dir, repository)):
-                os.makedirs(name=os.path.join(model_dir, repository), exist_ok=True)
-            snapshot_download(
-                token=hf_token,
-                repo_id=repository,
-                revision=release,
-                local_dir=os.path.join(model_dir, repository),
-            )
-        else:
-            hf_hub_download(
-                token=hf_token,
-                repo_id=repository,
-                revision=release,
-                filename=filename,
-                local_dir=model_dir,
-            )
-    except Exception as exc:
-        click.secho(
-            f"Downloading model failed with the following Hugging Face Hub error: {exc}",
-            fg="red",
-        )
-        raise click.exceptions.Exit(1)
-
-
-class TorchDeviceParam(click.ParamType):
-    """Parse and convert device string
-
-    Returns a torch.device object:
-    - type is one of 'cpu', 'cuda', 'hpu'
-    - index is None or device index (e.g. 0 for first GPU)
+    If this is your first time running ilab, it's best to start with `ilab init` to create the environment.
```

Comment:
Another spot where it should say `ilab config init` instead of `ilab init`

Reply:
https://github.com/instructlab/instructlab/issues/1331